### DISCUSSION TITLE
Allow ability to add adjustments before order is marked as complete

### DISCUSSION THREAD

**Discussion by engram-design** [craftcms/commerce#2340] [Ideas]
I can't seem to create order adjustments for the overall order or line item, unless I've marked the order as completed. 

Should this be an option?

**Comment by peteeveleigh** [comment]
Is this intended behaviour or something which will change?

**Comment by lukeholder** [comment]
There are currently only 2 modes for an order to recalculate, when the order is a cart (isCompleted=false) then the cart is always in a recalculate_all mode, which would  lose your manual adjustments. 

When the order is complete, the recalculating mode defaults to recalculate_none, and any manual adjustments can be made and will not be lost (unless you click the recalculate all button at the bottom).

We are considering more recalculate adjustment modes for the order. Like recalculating the adjustments only, but not the line items.
